contrib/vexecutor/tuplebatch.c

```diff
@@ -150,19 +150,26 @@ tbSerialization(TupleBatch tb )
     return ret;
 }
 
-TupleBatch tbDeserialization(unsigned char *buffer)
+void tbDeserialization(unsigned char *buffer,TupleBatch* pTB )
```

Comment:
Is tuple batch deserialization always successful ? If not, please use bool for return value type.